blocks/editable/index.js

```diff
@@ -303,7 +303,7 @@ export default class Editable extends Component {
 			if ( isEmpty && this.props.onReplace ) {
 				// Necessary to allow the paste bin to be removed without errors.
 				setTimeout( () => this.props.onReplace( content ) );
-			} else {
+			} else if ( this.props.onSplit ) {
```

Comment:
Should this check be consolidated in `splitContent` instead, so in case we add another call to the function we don't need to similarly guard?